gsearch.py

```diff
@@ -16,13 +18,22 @@
 import re
 import random
 import types
+from dotenv import load_dotenv
 from bs4 import BeautifulSoup
 
 reload(sys)
 sys.setdefaultencoding('utf-8')
 
-base_url = 'https://www.google.com.hk/'
-results_per_page = 10
+# Load config from .env file
+# TODO: Error handling
+try:
+    dotenv_path = join(dirname(__file__), '.env')
+    load_dotenv(dotenv_path)
```

Comment:
I think it's better to use find_dotenv(), this function will try to find a .env file by (a) guessing where to start using file or the working directory - allowing this to work in non-file contexts such as IPython notebooks and the REPL, and then (b) walking up the directory tree looking for the specified file – called .env by default